patterns/templates/jolla-configuration-@DEVICE@.yaml

```diff
@@ -4,8 +4,25 @@ Requires:
 - pattern:jolla-hw-adaptation-@DEVICE@
 - pattern:jolla-sailfish-applications
 - pattern:jolla-ui-wayland
+- pattern:jolla-ui-configuration-z@ICON_RES@
```

Comment:
This should be enough here: sailfish-content-graphics-z@ICON_RES@ the pattern will be dropped in future.